system/security/LdapSecurity/ldapconnection.cpp

```diff
@@ -4688,7 +4702,33 @@ class CLdapClient : implements ILdapClient, public CInterface
     }
 
 private:
+    class SDServerCtlWrapper
+    {
+    public:
+        LDAPControl **ctls = nullptr;
+        LDAPControl* ctl = nullptr;
+        StringBuffer oidbuf, valbuf;
 
+        SDServerCtlWrapper(bool isAzureAD)
```

Comment:
isAzureAD never actually seems to be checked.  Is this backward compatible?

Reply:
Meant to check it, guess forgot. Will add...